data/mt/retired/Wendy-Mckamey-8f827a3b-16fe-4e09-bad0-c3e29c73d9c8.yml

```diff
@@ -3,22 +3,30 @@ name: Wendy Mckamey
 given_name: Wendy
 family_name: Mckamey
 gender: Female
-email: Wendy.McKamey@mtleg.gov
+email: Wendy.McKamey@legmt.gov
 party:
 - name: Republican
 roles:
 - end_date: 2016-12-31
   type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:mt/government
   district: '23'
-- type: lower
+- end_date: '2022-12-31'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:mt/government
   district: '19'
+- type: upper
```

Comment:
@bfossen-ce this person starts in the senate on 01/02/2023, do they need a start date?